contents/handbook/growth/sales/why-buy-posthog.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Why buy PostHog
+sidebar: Handbook
+showTitle: true
+---
+
+# AKA our Value Proposition
+
+## A single place to view all of your user data
+
+With disparate tools, it's hard to get an understanding of your users both individually and at an aggregate level.  By integrating a range of products into one platform we provide a greater picture of our customer's users.  Common cross-product use cases are:
+
+ - When looking at a conversion funnel, click a dropped off segment of users to view their session replays.  This helps you understand potential friction/interface issues. 
+ - After launching an experiment, click to see the session replays of users experiencing a specific variant, showing you whether your users are experiencing it as intended.
+ - Create a survey tied to an experiment (via the feature flag), letting you gather qualitative feedback from users who are experiencing this new feature for the first time.
+ - Attach revenue data stored in our data warehouse from Stripe to product analytics to see behaviors which drive revenue acquisition and growth.
```

Comment:
Are the more examples than this? (This is great, just feels like we could have 10+ ideally!)

Reply:
Yeah these are the ones that come to mind first but I agree there are some missing - shall we get this in and then encourage folks to contribute their ideas?

Reply:
Sounds good - I bet @camerondeleone has a bunch in his head 👁️ 👁️